platform/source/LocalFileSystem.cpp

```diff
@@ -14,6 +14,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+#if defined(__ARMCC_VERSION)
```

Comment:
In commit "semihosting: add compat header where is required",

```diff
- explictely
+ explicitly
```

Reply:
fixed

Reply:
The spelling error in the commit message is still present.